Makefile

```diff
@@ -4,26 +4,39 @@ eslint:
 stylelint:
 	npm run stylelint
 
+build-test:
+	npm install
+	npm run build
```

Comment:
The deleted version from .travis.yml is `node run build` instead of `npm run build`. What is the difference between the two?